docs/USER_GUIDE.md

```diff
@@ -0,0 +1,27 @@
+# User Guide
+
+## Postgres
+
+This project relies upon several Postgres features that require elevated privileges.
+If you are configuring this in an environment where you do not possess these privileges,
+please work with your DBA to configure a new database in the following way:
+
+```sql
+-- Create a new database to store assets and relations.
+CREATE SCHEMA assetdb;
```

Comment:
change to database

Reply:
likely need to create a schema, update default schema for user if postgres 15 (`public` schema is no longer accessible by standard users / roles).